MediaGalleryRenditions/Test/Integration/Model/ResourceModel/GetAssetIdsByContentFieldTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ *
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryRenditions\Test\Integration\Model\ResourceModel;
+
+use Magento\Framework\Exception\InvalidArgumentException;
+use Magento\MediaContentApi\Api\GetAssetIdsByContentFieldInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * GetAssetIdsByContentFieldTest for MediaGalleryRenditions
+ */
+class GetAssetIdsByContentFieldTest extends TestCase
```

Comment:
Let's avoid using entities fixtures by using the integration test similar to `\Magento\MediaContent\Model\ExtractAssetsFromContentTest` for testing renditions